docs/retrieval-augmented-generation/rag-repo.md

```diff
@@ -0,0 +1,110 @@
+# New repository for RAG
+
+| Created  | Dec 5, 2024 |
+| -------- | -------- |
+| Authors | Bill Murdock |
+| Replaces | N/A |
+| Replaced by | N/A |
+
+## What
+
+We want a retrieval-augmented generation (RAG) capability that provides outstanding results with minimal effort, is seamlessly integrated with InstructLab, and is also general enough to be used in other applications as well.
+
+## Why
+
+Many InstructLab users want to train a model and then use it to RAG.  Often they build something simple themselves for this purpose.  Two problems with this approach:
+
+- Building their own RAG is extra work.
+- Users who are not experts on RAG might not build a RAG that provides outstanding results.
+
+There is a very simple RAG capability at https://github.com/instructlab/rag .  It is not tightly integrated with InstructLab and it does not use any advanced RAG capabilities.  However, it might have some existing users so we can't just unilaterally delete it or replace it with something radically different.
```

Comment:
I would challenge this assumption - I'm seeing 3 commits from 3 months ago by @aldopareja, I severely doubt there are any users here, unless someone has evidence otherwise? https://github.com/instructlab/rag/commits/main/

IMO, it makes more sense to use an existing, well-named repository than to create a second one.

Reply:
+1.

Reply:
FWIW, I would also prefer this if we can make the internal politics of it work somehow.

Reply:
not sure i know about the internal politics that are blocking this somehow but on behalf of Aldo, I can safely say, please feel free to use it as you see fit :)